berserk/types/core/aliases.py

```diff
@@ -0,0 +1,7 @@
+"""Aliases for common types"""
+from __future__ import annotations
+
+from typing_extensions import TypeAlias
+
+ID: TypeAlias = str
+Username: TypeAlias = str
```

Comment:
I don't think those aliases are necessary, and make the code more indirect, `str` is fine

Reply:
Fine, will fix.